src/components/Wallet/WalletConnectProvider.tsx

```diff
@@ -13,12 +14,13 @@ import {
   WalletMobileProvider,
   XDEFICosmosExtensionProvider,
 } from '@delphi-labs/shuttle-react'
-import { FC } from 'react'
+import { FC, useEffect, useState } from 'react'
 
 import chains from 'chains'
 import { LocalStorageKeys } from 'constants/localStorageKeys'
 import { WALLETS } from 'constants/wallets'
 import { WalletID } from 'types/enums'
+import AppKitProvider from './AppKitProvider'
```

Comment:
Avoid relative imports

```suggestion
import AppKitProvider from 'components/Wallet/AppKitProvider'
```